code/datums/uplink/highly visible and dangerous weapons.dm

```diff
@@ -49,6 +49,21 @@
 	item_cost = 14
 	path = /obj/item/weapon/gun/projectile/automatic/tommygun
 
+/datum/uplink_item/item/visible_weapons/landmine
+	name = "Land Mine"
+	item_cost = 5
+	path = /obj/item/weapon/landmine
+
+/datum/uplink_item/item/visible_weapons/emplandmine
+	name = "EMP Land Mine"
+	item_cost = 4
+	path = /obj/item/weapon/landmine/emp
+
+/datum/uplink_item/item/visible_weapons/sleepylandmine
+	name = "N2o Land Mine"
```

Comment:
This should be stylized as `N2O`, not `N2o`.

Reply:
Fixed.